Modules/Forum/classes/class.ilForumXMLParser.php

```diff
@@ -26,7 +26,25 @@ class ilForumXMLParser extends ilSaxParser
     protected $schema_version = null;
 
     private $db;
-
+    private ilObjUser $aobject;
+    private $importDirectory;
+    private array $forumArray;
+    private array $postArray;
+    private array $threadArray;
+    private string $cdata;
+    private array $contentArray;
+    /**
+     * @var int
+     */
+    private int $forum_obj_id = 0;
+    private int $frm_last_mapped_top_usr_id = 0;
+    private mixed $lastHandledForumId;
```

Comment:
We MUST NOT use the explicit `mixed` type. The code MUST be compatible with PHP 7.4.